cves/kernel/CVE-2017-1000252.yml

```diff
@@ -231,7 +231,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: virt
   note:
```

Comment:
How was the subsystem determined? An explanation needs to be provided.

Reply:
Thanks! will update